src/components/Board.js

```diff
@@ -16,18 +25,95 @@ class Board extends Component {
     };
   }
 
+  componentDidMount() {
+
+    let boardURL = `${url}${this.props.boardName}/cards`
+
+    axios.get(boardURL)
+      .then((response) => {
+
+        this.props.updateStatusCallback('successfully loaded cards', 'success');
+
+        let cardInfo = response.data.slice(0,100);
+
+
+        let cards = [];
+
+        let i;
+        for (i = 0; i < cardInfo.length; i++) {
+          let card = {};
+
+          card["id"] = cardInfo[i].card.id
+          card["text"] = cardInfo[i].card.text
+          card["emoji"] = cardInfo[i].card.emoji
+
+          cards.push(card);
+        }
+
+        this.setState({cards: cards});
+      })
+      .catch((error) => {
+
+        this.props.updateStatusCallback(error.message, 'error');
+      });
+  }
+
+  deleteCard = (cardID) => {
+
+    let deleteURL = `${url}${this.props.boardName}/cards/${cardID}`;
+    axios.delete(deleteURL);
+
+
+    let cards = this.state.cards;
+
+    for (let i = 0; i < cards.length; i++) {
+      if (cards[i].id === cardID) {
+        cards.splice(i, 1)
+
+        this.setState({cards: cards});
+      }
+    }
+  }
+
+  addCard = (card) => {
+    let boardURL = `${url}${this.props.boardName}/cards`
+    axios.post(boardURL, card);
```

Comment:
Same as with the delete request, we should have `then` and `catch` handlers for this.

In particular, its important to have the following code (where we insert the new card into the state) only happen in the `then` handler, because we don't want to add the card "locally" if the API actually rejected our request.